pkg/manager/member/startscript/v1/template.go

```diff
@@ -550,9 +550,21 @@ echo "/dm-master ${ARGS}"
 exec /dm-master ${ARGS}
 `))
 
+// TODO: refactor to confine the checking script within the package
+var DMMasterCheckDNSV1 string = `
+digRes=$(dig ${domain} A ${domain} AAAA +search +short 2>/dev/null)
```

Comment:
why output to `/dev/null`?

Reply:
Just follow the convention for `nslookup` to silent the stderr output:
https://github.com/pingcap/tidb-operator/blob/4e9e294ead1a935e7ac5a60f193ba50f25450d06/pkg/manager/member/startscript/v1/template.go#L511

Reply:
OK. but do not silent the stderr may be useful.